pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -619,6 +620,9 @@ def parse_args_and_make_package(args=None):
     ap.add_argument('--service', dest='services', action='append', default=[],
                     help='Declare a new service entrypoint: '
                          'NAME:PATH_TO_PY[:foreground]')
+    ap.add_argument('--native-service', dest='native_services', action='append', default=[],
+                    help='Declare a new native service: '
+                         'package.name.service')
```

Comment:
Minor: no sure why this line is broken down in two

Reply:
No specific reason, I just used the same codestyle than for the previous argument (line 620/621) for consistency.

Thanks for the merge!